### PR TITLE
test: Add readme test and update readme

### PR DESCRIPTION
**Pull request by ZuLiangWang**
# Which issue does this PR close?

# Rationale for this change
Update readme file and add runnable java code for readme.

# What changes are included in this PR?
Readme file details and test java code.

# Are there any user-facing changes?
all user used this client.

# How does this change test
Pass the CI

### CODE REVIEW

File: ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ * @author kesheng
+ * @date 2022/8/9 5:52 下午
+ */
+public class ReadmeTest {
+
+    @Ignore
+    @Test
+    public void readmeTest() throws ExecutionException, InterruptedException {
+        // CeresDBx options
+        final CeresDBxOptions opts = CeresDBxOptions.newBuilder("127.0.0.1", 8831) // CeresDB 默认配置端口号为 8831，不需要修改
```
Comment (jiacai2050) [inline ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java]: It's better to add comments in English

**Review by jiacai2050 — CHANGES_REQUESTED** [review]

**Comment by jiacai2050** [comment]
@ZuLiangWang Please sign CLA

File: ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java (hunk 1 above)
Comment (ZuLiangWang) [reply ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java]: fix

File: ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import org.junit.Test;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.stream.Collectors;
+
+/**
+ * @author kesheng
+ * @date 2022/8/9 5:52 下午
```
Comment (jiacai2050) [inline ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java]: - https://github.com/CeresDB/ceresdb-client-java/blob/4db4671c88c2d5675da7dcfb692d9be10101e3f9/ceresdb-http/src/main/java/com/ceresdb/http/HttpManagementClient.java#L53

I check other java files, only author is included, maybe we should keep the same format?

**Review by jiacai2050 — CHANGES_REQUESTED** [review]

File: ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        if (!client.init(opts)) {
+            throw new IllegalStateException("Fail to start CeresDBxClient");
+        }
+
+        final long t0 = System.currentTimeMillis();
+        final long t1 = t0 + 1000;
+        final long t2 = t1 + 1000;
+        final Rows data = Series.newBuilder("machine_metric").tag("city", "Singapore").tag("ip", "127.0.0.1")
+                .toRowsBuilder()
+                // codes below organizes 3 lines data (3 timestamps) for the `cpu` and `mem` column, this will just transport once through network. CeresDB encourage practices like this, because the SDK could use efficient compression algorithm to reduce network overhead and also be friedly to the sever side.
```
Comment (jiacai2050) [inline ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java]: typo: friendly

Comment (jiacai2050) [inline ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java]: reduce network overhead -> reduce network traffic

File: ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java (hunk 2 above)
Comment (ZuLiangWang) [reply ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java]: fix, now the format is consistent with the original

File: ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java (hunk 3 above)
Comment (ZuLiangWang) [reply ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java]: fix

Comment (ZuLiangWang) [reply ceresdb-example/src/test/java/com/ceresdb/ReadmeTest.java]: fix

**Review by jiacai2050 — APPROVED** [review]

**Pull request merged**